microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -218,4 +218,13 @@ Issued: 2023-07-26
 
 {product-title} release 4.13.6, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4428[RHBA-2023:4428] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:4426[RHSA-2023:4426] advisory.
 
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
```

Comment:
I see that it's like this throughout the file (not just this PR), but should the word "image" be marked up here? Seems like the following would make sense:

```suggestion
For the `TopoLVM` image, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
```

If so, suggest changing throughout, either in this or a follow-up.

Reply:
Oh yes, somehow along the way the image must have been pulled out of the backtick by accident. I will fix in this PR. Thanks for catching @adellape